cmd/nginx-ingress/main.go

```diff
@@ -243,6 +242,25 @@ func validateIngressClass(kubeClient kubernetes.Interface) {
 	}
 }
 
+func checkNamespaces(kubeClient kubernetes.Interface) {
+	if *watchNamespaceLabel != "" {
+		// bootstrap the watched namespace list
+		var newWatchNamespaces []string
+		nsList, err := kubeClient.CoreV1().Namespaces().List(context.TODO(), meta_v1.ListOptions{LabelSelector: *watchNamespaceLabel})
+		if err != nil {
+			glog.Warningf("Error when getting Namespaces with the label selector %v: %v", watchNamespaceLabel, err)
+		}
```

Comment:
I think we can remove the word "Error" from this log since it's a warning log. Could be changed to "unable to get Namespaces with the label selector"

Reply:
Changed it to an error message